core: start admission controller without any script (#9934)
rook/rook: Documentation/admission-controller-usage.md

> ## Quick Start

Reviewer: How about if we remove this topic, and create a new section in the pre-requisites (pre-reqs.md) page? I'm thinking after the Ceph prereqs section, we would have a new section called `## Admission Controller`. We would explain that it's recommended, but optional to first install the cert manager to enable the admission controller.

Author: done

> To deploy the Rook admission controllers just in cert manager:

Reviewer: ```suggestion
To deploy the Rook admission controllers, ensure the cert manager is enabled:
```

Author: done
